pkgs/development/libraries/quantlib/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, boost
+}:
+
+stdenv.mkDerivation rec {
+  pname = "quantlib";
+  version = "1.29";
+
+  outputs = ["lib" "dev" "out" "bin"];
```

Comment:
```suggestion
  outputs = [ "dev" "lib" "out" ];
```

> error: builder for '/nix/store/bmn5nknwvaf0n4cjj69bxkl637cf5sqm-quantlib-1.29.drv' failed to produce output path for output 'bin' at '/nix/store/bmn5nknwvaf0n4cjj69bxkl637cf5s
qm-quantlib-1.29.drv.chroot/nix/store/03ckjcp24qa64fg6k1jjaywi79920ghl-quantlib-1.29-bin'

Reply:
I was in a bit of a hurry, I pushed it without checking, I had already fixed it

Reply:
I was in a bit of a hurry, I pushed it without checking, I had already fixed it
By the way, the code you gave will fail on my machine

Reply:
I think you forgot `lib`

Reply:
sorry, my bad, I didn't work, you can remove `lib` 🤦

Reply:
ok I removed it